azerothcore/azerothcore-wotlk
# fix(Core/Auras): SPELL_AURA_MOD_SCALE should scale additively instead…

… of multiplicatively

Co-authored-by: Shauren <shauren.trinity@gmail.com>



## Changes Proposed:
-  Make AURA_MOD_SCALE additive instead of multiplicative.

## Issues Addressed:

- Closes -

## SOURCE:


## Tests Performed:

- Tested in-game with #14426

## How to Test the Changes:


1. Check pet scale, they should be normal
2. Load the PR linked above, mushrooms from the boss should be fairly visible compared without this PR.
3. Check others creatures failing with scale?
4.
